### PR TITLE
Query and Update Database

### PR DESCRIPTION
**Pull request by principlebrothers**
### Complete milestone 3: 
- [x] Use primary key & foreign key mechanism for joining tables.
- [x] Query multiple tables.
- [x] Prepare complex queries that answer analytical questions.

### Add screenshot of queries and results for this milestone
query_one
query_two

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -6,4 +6,38 @@ CREATE TABLE animals (
...
+ADD
+    FOREIGN KEY (species_id) REFERENCES species(id) ON DELETE CASCADE;
+
+-- Add column owner_id which is a foreign key referencing the owners table
+ALTER TABLE
+    animals
+ADD
+    COLUMN owner_id INT,
+ADD
+    FOREIGN KEY (owner_id) REFERENCES owners(id) ON DELETE CASCADE;
```
Comment (mateo951) [inline schema.sql:43]: Well done adding the new tables and altering animals table ✔

File: schema.sql [hunk 2]
```diff
@@ -6,4 +6,38 @@ CREATE TABLE animals (
     escape_attempts INT NOT NULL,
     neutered BOOLEAN NOT NULL,
```
Comment (mateo951) [inline schema.sql:7]: Please be sure to add the new queries into your queries.sql file as it is part of the requirements 🤓

**Review by mateo951 — CHANGES_REQUESTED** [review]
### Hi @principlebrothers, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

File: schema.sql (hunk 2 above)
Comment (principlebrothers) [reply schema.sql:7]: Hello @mateo951, thanks for the reviewed feedback. There is one thing I want to point out from the documentation, and I believe you see my perspective. Inasmuch as I would implement what you have asked, from the attached image, it was not explicitly stated we add the queries to the` queries.sql.` 
feedback

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @principlebrothers, 

### Good job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can get to me on slack  using this handle @mihndim.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by principlebrothers** [comment]
Thank you, @Mihndim2020.

**Pull request merged**